modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/Jvm.kt

```diff
@@ -67,7 +67,7 @@ open class Jvm constructor(
             return toolsJar
         }
         if (javaHome!!.name.equals("jre", true)) {
-            javaHome = javaHome!!.parentFile
+            _javaHome = javaHome!!.parentFile
```

Comment:
Can you explain why this works?

Reply:
`javaHome` is a shadow of `_javaHome`, it __always__ returns the value of `_javaHome`. Hence the need to set `_javaHome` directly.

Simple commented example:

[Run](https://try.kotlinlang.org/#/UserProjects/au6japk87o09cfmsdhruoj13vl/sfe79i5pj8e95v17017j53lc56)
```kotlin
open class Example {
    private var _a: String = "_a"
    
    var a: String = "a"
    	get() = _a // always return _a
    
    // Change the value of _a
    fun change_a(newValue: String) {
        _a = newValue
    }
}

fun main(args: Array<String>) {
    val example = Example()
    
    example.a = "something" // does nothing, since getting "a" always returns the value of "_a"
    println(example.a) // prints: "_a"

    example.change_a("something else") // sets the value of "_a"
    println(example.a) // prints: "something else"
}
```

Make sense?